tensorrt_llm/inputs/__init__.py

```
@@ -2,13 +2,14 @@
 from .data import PromptInputs, TextPrompt, TokensPrompt, prompt_inputs
 from .evs import compute_retained_tokens_count, compute_retention_mask
 from .multimodal import MultimodalInput
-from .registry import (BaseMultimodalDummyInputsBuilder,
-                       BaseMultimodalInputProcessor, ExtraProcessedInputs,
+from .registry import BaseMultimodalDummyInputsBuilder  # yapf: disable
+from .registry import support_multimodal_disaggregated  # yapf: enable
```

Ruff (0.15.10)

[warning] This suppression comment is invalid because it cannot be at the end of a line

Remove this comment

(RUF028)